components/breadcrumbs/breadcrumbs.stories.js

```diff
@@ -1,6 +1,5 @@
 import { createTemplateFromVueFile } from '@/common/storybook_utils';
```

Comment:
in this file, we have a description here: https://github.com/dialpad/dialtone-vue/pull/1246/files#diff-88d4eddbc5b117284ff3294a480b8891ac689e6d85117ea205b5d8f4f002f10aR50

Where the object contains url instead of href which is misleading a bit. 
```
summary: '{ url: string, label: string }[]',
```

Reply:
thanks, definitely outdated.